integration-tests/parser/import/test.test.js

```diff
@@ -0,0 +1,10 @@
+const test = require('ava');
+const {exec} = require('../../setup');
+
+test(
+    'cli: able to have two source directories',
```

Comment:
This test case seems to say what the other one should have said; if that's the case and they're switched accidentally, what should this one be about?

Reply:
```suggestion
    'parser: able to import variable from module',
```

oops!